seed/data_importer/match.py

```diff
@@ -123,10 +123,11 @@ def match_and_link_incoming_properties_and_taxlots(file_pk, progress_key):
 
         # Within the ImportFile, merge -States together based on user defined matching_criteria
         log_debug('Start Properties inclusive_match_and_merge')
-        promoted_property_ids, property_merges_within_file_count = inclusive_match_and_merge(promoted_property_ids, org, PropertyState)
+        promoted_property_ids, property_merges_within_file_count = inclusive_match_and_merge(promoted_property_ids, org, PropertyState, progress_data)
 
         # Filter Cycle-wide duplicates then merge and/or assign -States to -Views
         log_debug('Start Properties states_to_views')
+        progress_data.step('Assigning Properties to Views')
         merged_property_views, property_duplicates_against_existing_count, property_new_count, property_merges_against_existing_count, property_merges_between_existing_count = states_to_views(
```

Comment:
I think we might need to add "sub progress" stuff for this one as well -- it's where we actually search the entire database for matching properties and merge them. So this will be particularly slow when there are many thousands of properties already in the database.